apps/web-giddh/src/app/invoice/preview/models/invoice-preview-details/invoice-preview-details.component.ts

```diff
@@ -809,7 +809,7 @@ export class InvoicePreviewDetailsComponent implements OnInit, OnChanges, AfterV
         this.invoiceSearch = term;
         this.invoiceSearchEvent.emit(this.invoiceSearch);
         this.filteredData = this.items.filter(item => {
-            return item.voucherNumber.toLowerCase().includes(term.toLowerCase()) ||
+            return item.voucherNumber?.toLowerCase()?.includes(term?.toLowerCase()) ||
                 item.account.name.toLowerCase().includes(term.toLowerCase()) ||
```

Comment:
```suggestion
                item.account.name?.toLowerCase().includes(term?.toLowerCase()) ||
```

Reply:
Sir I am getting the value of item.account.name but voucherNumber is getting null that's why Exception is Occuring
that's Why I added the SafeNavigation Operator there Only